### PR TITLE
Add JetBrains toolbox manifest

### PR DESCRIPTION
**Pull request by cfanoulis**
Adds a manifest for Jetbrain's [Toolbox](https://www.jetbrains.com/toolbox-app/) app.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/318)

### CODE REVIEW

**Comment by denelon** [comment]
Are there silent switches that can help this to run unattended?

**Comment by cfanoulis** [comment]
Yup, will add in a bit

**Comment by cfanoulis** [comment]
This... is the way JetBrains actually suggests installing the app.

I will contact JetBrains technical support tomorrow and ask again.

**Comment by vasilvestre** [comment]
My PR #894 works for it. Take my code if you want.

File: manifests/JetBrains/Toolbox/1.17.6856.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Name: JetBrains Toolbox
+Publisher: JetBrains
+License: Copyright (c) JetBrains s.r.o.
+AppMoniker: toolbox
+Tags: jetbrains, toolbox, download
+Description: A suite of professional development tools for a range of languages and technologies
+Homepage: https://www.jetbrains.com/toolbox-app/
+Switches:
+  Silent: /S
+  InstallLocation: /D=<INSTALLPATH>
```
Comment (ForNeVeR) [inline manifests/JetBrains/Toolbox/1.17.6856.yaml:12]: It is possible that quotes are required here.
```suggestion
  InstallLocation: /D="<INSTALLPATH>"
```

Other approved manifest samples I found in winget use quotes around `<INSTALLPATH>`.

**Comment by superusercode** [comment]
can't this manifest just use the nullsoft installertype and not have to use the installlocation or the switches?

**Comment by CoolTomatos** [comment]
Any update on this?

**Comment by cfanoulis** [comment]
Whoops, sorry about that. Slipped off my agenda, and forgot about this.

I'll probably try to set the installer as type Nullsoft to see if that
fixes the issue.

**Comment by KevinLaMS** [comment]
Application fails to run.  it shows error 
Failed to load jvm.dll %1 is not a valid win32 application.

**Pull request closed without merging**